HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -4508,7 +4508,7 @@ Patch "Always Play 'One Way Dream' During Credits" in "Audio" by "M&M"
     );
 }
 
-Patch "Always Play 'Vandalize' During Credits" in "Audio" by "M&M"
+Patch "Always Play 'Vandalize' During Credits" in "Audio" by "M&M" does "Plays 'Vandalize' in place of 'One Way Dream' during the credits in Hard mode."
```

Comment:
- Change "in place of" to "instead of".
- Change "in Hard mode" to "on Hard difficulty".